### ISSUE TITLE
Is it possible to execute Test::More with B::C?

### ISSUE THREAD

**Issue by EvanCarroll** [rurban/perl-compiler#436]
I'm trying to run the testing suite for a module found on cpan, such as `Syntax::Keyword::Try`. It would be nice if this was possible but i'm getting failure with anything that uses `Test::More`.

https://stackoverflow.com/questions/66231953/how-can-i-test-if-a-module-is-compatible-with-the-perl-compiler-perlcc

**Comment by rurban** [comment]
Yes, we ran the full core testsuite and most used modules with B::C. See https://github.com/perl11/p5-coretests for the infrastructure.

How recent perl's broke it, and they were unwilling to fix their regressions.
